src/devdox_ai_git/git_managers.py

```
@@ -402,6 +771,399 @@ def get_user_repositories(
+
+        Raises:
+            DevDoxGitException: If repository deletion fails
+        """
+        try:
+            # Get project object if ID provided
+            if isinstance(project_or_id, int):
+                project = self._git_client.projects.get(project_or_id, timeout=timeout)
+                project_name = f"Project ID: {project_or_id}"
+            else:
+                project = project_or_id
+                project_name = project.name
```

Ruff (0.13.2)

Local variable `project_name` is assigned to but never used

Remove assignment to unused variable `project_name`

(F841)